src/main/java/seedu/address/ui/NavigationButtonPanel.java

```diff
@@ -0,0 +1,46 @@
+package seedu.address.ui;
+
+import javafx.fxml.FXML;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.StackPane;
+
+
+/**
+ * The Main Window. Provides the basic application layout containing
```

Comment:
Perhaps you could make the docs here neater? Let a period be the end of a line.

Reply:
This is not the end of a sentence so there's no need for a period